Index: steam-1.0.0.18/usr/bin/steam
===================================================================
--- steam-1.0.0.18.orig/usr/bin/steam
+++ steam-1.0.0.18/usr/bin/steam
@@ -178,7 +178,7 @@ function install_bootstrap()
 	umask "$omask"
 
 	# Try and install any extra packages
-	install_extra_packages jockey-common python-apt
+	# install_extra_packages jockey-common python-apt
 }
 
 function repair_bootstrap()
Index: steam-1.0.0.18/usr/share/applications/steam.desktop
===================================================================
--- steam-1.0.0.18.orig/usr/share/applications/steam.desktop
+++ steam-1.0.0.18/usr/share/applications/steam.desktop
@@ -1,42 +1,9 @@
 [Desktop Entry]
 Name=Steam
 Comment=Application for managing and playing games on Steam
-Exec=/usr/bin/steam %U
+Exec=steam %U
 Icon=steam
 Terminal=false
 Type=Application
 Categories=Network;FileTransfer;Game;
-MimeType=x-scheme-handler/steam
-Actions=Store;Community;Library;Servers;Screenshots;News;Settings;Friends
-
-[Desktop Action Store]
-Name=Store
-Exec=steam steam://store
-
-[Desktop Action Community]
-Name=Community
-Exec=steam steam://url/SteamIDControlPage
-
-[Desktop Action Library]
-Name=Library
-Exec=steam steam://open/games
-
-[Desktop Action Servers]
-Name=Servers
-Exec=steam steam://open/servers
-
-[Desktop Action Screenshots]
-Name=Screenshots
-Exec=steam steam://open/screenshots
-
-[Desktop Action News]
-Name=News
-Exec=steam steam://open/news
-
-[Desktop Action Settings]
-Name=Settings
-Exec=steam steam://open/settings
-
-[Desktop Action Friends]
-Name=Friends
-Exec=steam steam://open/friends
+MimeType=x-scheme-handler/steam;
